packages/legend-application-query/src/components/QueryEditor.tsx

```diff
@@ -360,6 +365,111 @@ const QueryEditorExistingQueryHeader = observer(
+      event,
+    ) => {
+      setQueryVersionId(event.target.value);
+    };
+
+    const updateQueryVersionId = applicationStore.guardUnhandledError(
+      async (): Promise<void> => {
+        flowResult(
+          existingEditorStore.updateState.updateQuery(
+            undefined,
+            queryVersionId,
+          ),
+        ).catch(applicationStore.alertUnhandledError);
+      },
+    );
+
+    const closeModal = (): void => updateState.setShowQueryInfo(false);
+
+    return (
+      <Dialog
+        open={updateState.showQueryInfo}
+        onClose={closeModal}
+        classes={{ container: 'search-modal__container' }}
+        PaperProps={{ classes: { root: 'search-modal__inner-container' } }}
+      >
+        <Modal
+          darkMode={
+            !applicationStore.layoutService.TEMPORARY__isLightColorThemeEnabled
+          }
+          className="search-modal"
+        >
+          <ModalTitle title="Query Info" />
+          <Panel>
+            <PanelFullContent>
+              <div className="query-preview__field">
+                <div className="query-preview__field__label">Project</div>
+                <div className="query-preview__field__value">
+                  {generateGAVCoordinates(
+                    query.groupId,
+                    query.artifactId,
+                    undefined,
+                  )}
+                </div>
+              </div>
+              <div className="query-preview__field">
+                <div className="query-preview__field__label">Mapping</div>
+                <div className="query-preview__field__value">
+                  {query.mapping.value.name}
+                </div>
+              </div>
+              <div className="query-preview__field">
+                <div className="query-preview__field__label">Runtime</div>
+                <div className="query-preview__field__value">
+                  {query.runtime.value.name}
+                </div>
+              </div>
+              <div className="query-preview__field">
+                <div className="query-preview__field__label">Version</div>
+                <div className="query-preview__field__input">
+                  <InputWithInlineValidation
+                    className="panel__content__form__section__input"
+                    value={queryVersionId}
+                    spellCheck={false}
+                    onChange={changeQueryVersionId}
+                    placeholder="Query Version"
+                    showEditableIcon={true}
+                  />
+                  <button
+                    className="btn btn--dark query-preview__field__input__action"
+                    onClick={updateQueryVersionId}
+                  >
+                    save
+                  </button>
+                </div>
+              </div>
+              {query.owner && (
+                <div className="query-preview__field">
+                  <div className="query-preview__field__label">owner</div>
```

Comment:
Should this be capitalized like the other labels?
```suggestion
                  <div className="query-preview__field__label">Owner</div>
```